pyproject.toml

```diff
@@ -42,6 +42,7 @@ pyyaml = "*"
 starlette = "*"
 typing-extensions = "*"
 requests = { version = "*", optional = true }
+maturin = "^0.12.20"
```

Comment:
Shouldn’t it be a development dependency?

Reply:
it should

Reply:
fixed